script/upgrades/MU01/governance/Config.sol

```diff
@@ -354,4 +370,74 @@ library Config {
       config.minimumReports = 2;
     }
   }
+
+  function cEURUSDCConfig(
```

Comment:
do not review pool configs in this pr

Reply:
they should be reviewed in Nadiem's pr. There was no way for me to work on this unless I pulled Nadiem's branch. I've stated in the description to not merge this before his pr is merged and I'd pull the latest so there'd be no diff after. I hope this clears up the confusion.